tests/introspection/tests.py

```diff
@@ -53,6 +53,12 @@ def test_table_names_with_views(self):
         self.assertNotIn('introspection_article_view',
                          connection.introspection.table_names())
 
+    def test_unmanaged_through_model(self):
+        table_name = ArticleReporter._meta.db_table
+        connection.cursor().execute('CREATE VIEW {} AS SELECT 1 AS id'.format(table_name))
+        tables = connection.introspection.django_table_names(only_existing=True)
```

Comment:
I tested locally and confirmed the view creation is not necessary.

Make sure you don't pass `only_existing=True` to the `django_table_names` call and the test will fail without the fix applied and pass with it.